engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscriptions-helper.service.js

```diff
@@ -49,7 +49,7 @@ angular.module('Bastion.subscriptions').service('SubscriptionsHelper',
 
             selected = [];
             angular.forEach(table.getSelected(), function (subscription) {
-                selected.push({"id": subscription.id});
+                selected.push({"id": subscription.cp_id});
```

Comment:
So it turns out if you attach any amount of subscriptions to a content host and then do Katello::System.first.unsubscribe(nil), all the subscriptions will be removed from that content host. I added a check in the controller to prevent this now, but before it was returning nil on ```entitlement_id = @system.find_entitlement(subscription[:id])``` because subscription[:id] was not a candlepin id and passing that to unsubscribe.